### ISSUE TITLE
💻 default tags avoiding duplicate/spam tags

### ISSUE THREAD

**Issue by hasan-sh** [hedyorg/hedy#5657]
> Maybe we can make some tags, instead of letting the teachers create their own tags, because at the moment their are a lot of nonsense tags. This defeats the purpose of a tag.

This needs to be discussed and decided; so in another PR?

_Originally posted by @hasan-sh in https://github.com/hedyorg/hedy/issues/5475#issuecomment-2200622434_

**Comment by Felienne** [comment]
> > Maybe we can make some tags, instead of letting the teachers create their own tags, because at the moment their are a lot of nonsense tags. This defeats the purpose of a tag.
> 
> This needs to be discussed and decided; so in another PR?
> 
> _Originally posted by @hasan-sh in [#5475 (comment)](https://github.com/hedyorg/hedy/issues/5475#issuecomment-2200622434)_

Not a bad idea, but maybe we should first clean up the existing tags? (I think you yourself are able to do so as superteacher?) Because I think a lot of them are also created by us?

**Comment by Felienne** [comment]
Alternatively, we can make a propose-approve system but I think the cleanup absolutely needs to happen first!

**Comment by hasan-sh** [comment]
There are many tags named after keywords:

![image](https://github.com/hedyorg/hedy/assets/20051470/21536fbf-e43f-4569-8b3d-afb165128baf)



Should I remove these?

**Comment by Felienne** [comment]
> There are many tags named after keywords:
> 
> ![image](https://private-user-images.githubusercontent.com/20051470/346576286-21536fbf-e43f-4569-8b3d-afb165128baf.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA1MzQ4NjYsIm5iZiI6MTcyMDUzNDU2NiwicGF0aCI6Ii8yMDA1MTQ3MC8zNDY1NzYyODYtMjE1MzZmYmYtZTQzZi00NTY5LThiM2QtYWZiMTY1MTI4YmFmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzA5VDE0MTYwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc4NGVmNWE5NmI3YzFhZmZkMzgzN2U0YTU4YTAyZmU0NmYyNjNlM2E5NTJkMjE2MjE5M2VhNDFmYzkxYTQ4NGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ACznHISEjAXneIB6lG4iaRtviTN0TY5NaxH45Y5-T9Q)
> 
> What should we do about them? I'm now removing tags that make no sense or are only used in basic/default adventures!

As discussed in the meeting, let's keep these!
